app/assets/stylesheets/players.scss.erb

```diff
@@ -381,7 +381,7 @@ div {
   display: table;
 }
 
-.dps { display: table; }
+.calc { display: table; }
```

Comment:
.calc has already defined that it's a table so that isn't needed. but won't do any harm